ftfy/streamtester/__init__.py

```diff
@@ -17,14 +18,18 @@ def __init__(self):
         self.num_fixed = 0
         self.count = 0
 
-    def check_ftfy(self, text):
+    def check_ftfy(self, text, encoding_only=True):
         """
         Given a single text input, check whether `ftfy.fix_text_encoding`
         would change it. If so, display the change.
         """
         self.count += 1
+        text = unescape_html(text)
         if not possible_encoding(text, 'ascii'):
-            fixed = fix_text_encoding(text)
+            if encoding_only:
+                fixed = fix_encoding(text)
+            else:
+                fixed = fix_text(text, fix_character_width=False, uncurl_quotes=False)
```

Comment:
This is for that new stuff in accuracy.rst?  It might make sense for check_ftfy to just take kwargs to pass to fix_text, but if you're the only one who uses it you're probably just going to be manually changing this stuff anyway so it probably doesn't matter?